### ISSUE TITLE
Build failure with GHC 7.10

### ISSUE THREAD

**Issue by snoyberg** [agocorona/Workflow#5]
```
Unpacking to Workflow-0.8.1/
Resolving dependencies...
Configuring Workflow-0.8.1...
Building Workflow-0.8.1...
Preprocessing library Workflow-0.8.1...

on the commandline: Warning:
    -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS

Control/Workflow.hs:1:15: Warning:
    -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS
[1 of 4] Compiling Control.Workflow.Stat ( Control/Workflow/Stat.hs, dist/build/Control/Workflow/Stat.o )
[2 of 4] Compiling Control.Workflow ( Control/Workflow.hs, dist/build/Control/Workflow.o )

Control/Workflow.hs:206:10:
    Could not deduce (Applicative (WF s m))
      arising from the superclasses of an instance declaration
    from the context (Monad m)
      bound by the instance declaration at Control/Workflow.hs:206:10-36
    In the instance declaration for ‘Monad (WF s m)’
```

**Comment by agocorona** [comment]
Solved with the version 0.8.2 in Hackage.

Thanks

2015-04-30 8:55 GMT+02:00 Michael Snoyman <notifications@github.com>:

> Unpacking to Workflow-0.8.1/
> Resolving dependencies...
> Configuring Workflow-0.8.1...
> Building Workflow-0.8.1...
> Preprocessing library Workflow-0.8.1...
>
> on the commandline: Warning:
>     -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS
>
> Control/Workflow.hs:1:15: Warning:
>     -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS
> [1 of 4] Compiling Control.Workflow.Stat ( Control/Workflow/Stat.hs, dist/build/Control/Workflow/Stat.o )
> [2 of 4] Compiling Control.Workflow ( Control/Workflow.hs, dist/build/Control/Workflow.o )
>
> Control/Workflow.hs:206:10:
>     Could not deduce (Applicative (WF s m))
>       arising from the superclasses of an instance declaration
>     from the context (Monad m)
>       bound by the instance declaration at Control/Workflow.hs:206:10-36
>     In the instance declaration for ‘Monad (WF s m)’
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/agocorona/Workflow/issues/5>.
>



-- 
Alberto.

**Comment by snoyberg** [comment]
Thanks, I'll ping again if there are issues.
